### ISSUE TITLE
Provide App::perlimports to the image

### ISSUE THREAD

**Issue by bschmalhofer** [Perl/docker-perl-tester#60]
Hi,

I am fond of [perlimports](https://metacpan.org/pod/App::perlimports) for cleaning up my import statements. Would it make sense to add that module to the image? 

The list of dependencies looks a bit daunting though. Even worse, I tried to install App::perlimports in my Github action. This failed because DBB::ODBC could not be installed, likely because no ODBC driver manager was available.

**Comment by atoomic** [comment]
I see no objections to add it, but also note that you can add extra packages using a `cpanfile` with cpm or cpanm actions

Note: there is one example described here https://github.com/Perl/docker-perl-tester?tab=readme-ov-file#using-the-images-with-github-workflow
and the two actions are documented there:
* https://github.com/perl-actions/install-with-cpanm
* https://github.com/perl-actions/install-with-cpm

**Comment by oalders** [comment]
> I am fond of [perlimports](https://metacpan.org/pod/App::perlimports) for cleaning up my import statements

❤️ 

We could add it for perl >= '5.18.0'. It would pull in a lot of dependencies, but I'm not sure how bad that would be, considering how many dependencies these builds already contain.

**Comment by bschmalhofer** [comment]
> I see no objections to add it, but also note that you can add extra packages using a `cpanfile` with cpm or cpanm actions

Yes, that is what I do in https://github.com/RotherOSS/otobo/blob/5c899d77c7d9e33b9471ca26fcbfbc60a9ed68d1/.github/workflows/code_policy.yml#L60 .
This works fine, so adding App::perlimports would be just for convenience.
